src/generative_models/encoders.jl

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
empty line here?

Reply:
Interesting that the formatter didnt complain, thanks